test/operator/installation.go

```diff
@@ -25,6 +31,40 @@ func InstallFromSource(withCsi bool) features.Func {
 	return manifests.InstallFromFiles(paths)
 }
 
+func InstallViaMake() features.Func {
+	return func(ctx context.Context, t *testing.T, environmentConfig *envconf.Config) context.Context {
+		rootDir := getCorrectDir()
+
+		platform := kubeobjects.ResolvePlatformFromEnv()
+		makeTarget := "deploy"
+		if platform == kubeobjects.Openshift {
+			makeTarget = strings.Join([]string{makeTarget, "openshift"}, "/")
+		} else if platform == kubeobjects.Kubernetes {
+			makeTarget = strings.Join([]string{makeTarget, "kubernetes"}, "/")
+		}
+
+		err := exec.Command("make", "-C", rootDir, makeTarget).Run()
```

Comment:
Doesn't that also need the "build" and "manifests/branch" targets, or is having built the operator from the command line a mandatory pre-requisite one has to know?

Reply:
Yes, you're right, thanks. I'll fix this.